src/czechpmdevs/buildertools/commands/BlockInfoCommand.php

```diff
@@ -50,9 +46,8 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
             $item->setCustomName(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["name"]);
             $nbt = $item->getNamedTag();
             $nbt->setByte("buildertools", 1);
-            $nbt->setTag(Item::TAG_ENCH, new ListTag([], NBT::TAG_Compound));
             $item->setNamedTag($nbt);
-            /*$item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));*/
```

Comment:
I meant to remove the line. That custom enchantment should not be a thing anymore. I think

Reply:
I was try to implement it and it still work, its easier to see the distinguish between wand or block pick  stick and normal axe or stick

Reply:
I think custom enchantments should be used for item with special power. I just wanted to add blessing effect to the item. Also I would like to keep the similarity with 3.0.0 branch.